dotcms-integration/src/test/resources/META-INF/beans.xml

```diff
@@ -1,8 +1,7 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
For webapps the beans.xml is meant to be in the WEB-INF folder e.g. in dotCMS/src/main/webapp.   I can understand though if this caused problems in cases where we are not using this directly as a webapp, if this is the case maybe we do need it here.

Reply:
I think it is fine